src/Microsoft.Dnx.Runtime.Abstractions/Constants.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Dnx.Runtime
```

Comment:
We shouldn't expose anything like this publicly. It's an implementation detail that shouldn't be exposed. Especially until we're sure of these values.